static/js/reducers/commonUi_test.js

```diff
@@ -11,7 +11,7 @@ import {
   setEditVideoDesc,
   initEditVideoForm,
   showDialog,
-  hideDialog
+  hideDialog, showMenu, hideMenu
```

Comment:
nit: i think we should follow the one-per-line pattern

Reply:
OK